src/components/locations/constants.js

```diff
@@ -11,5 +11,15 @@ export const FILTER = 'FILTER';
 /** @constant */
 export const RELOAD = 'Reload';
 /** @constant */
+export const perWeek = '/week';
+/** @constant */
+export const naira = 'N';
```

Comment:
any reason why these constant names are not  uppercase

Reply:
will implement that